src/fswAlgorithms/imageProcessing/centerOfBrightness/_UnitTest/test_centerOfBrightness.py

```diff
@@ -99,8 +109,16 @@ def centerOfBrightnessTest(show_plots, image, blur, saveTest, valid, saveImage):
         moduleConfig.saveImages = True
     cob_ref = [input_image.width/2, input_image.height/2]
     if image == "half_half.png":
```

Comment:
:large_orange_diamond: By the looks of it, this entire conditional for image "half_half.png" can move down into the conditional with the assert at the end of this file. There's no usage that requires it here.

Reply:
Agreed